docs/baking.md

```diff
@@ -155,6 +155,13 @@ sudo -u tezos tezos-client bootstrapped
 By default `tezos-baking-<network>.service` will use the `baker` alias for the
 key that will be used for baking and endorsing.
 
+### Setting the Liquidity Baking toggle vote option
```

Comment:
There are two additional things that I think need to be mentioned in this section:
1) Once the default file is updated baking service needs to be restarted for changes to take effect
2) It's also possible to re-run `tezos-setup-wizard` and change vote there, and I think that this option is a bit more convenient than editing a file and restarting services manually